ports/espressif/supervisor/port.c

```diff
@@ -193,8 +220,12 @@ safe_mode_t port_init(void) {
 
     #ifdef CONFIG_SPIRAM
     if (esp_spiram_is_initialized()) {
-        size_t spiram_size = esp_spiram_get_size();
+        size_t spiram_size = spiram_size_usable_for_malloc();
```

Comment:
Does spiram_size_usable_for_malloc require you to let the IDF manage the spiram? Generally we have it ignore it.

Reply:
I renamed the function to avoid confusion. It has nothing to do with malloc, it's just the name of the function I copied in from esp-idf, where it was defined statically.